test/libpostalParser.js

```diff
@@ -130,6 +130,36 @@ tape('tests', function(test) {
 
   });
 
+  test.test('all known values should be adapted to pelias model', function(t) {
+    var node_postal_mock = function(query) {
+      t.equal(query, 'query value');
+
+      return [
+        {
+          component: 'city',
+          value: 'city value'
+        },
+        {
+          component: 'state',
+          value: 'state value'
+        }
+      ];
+    };
+
+    var parser = libpostalParser.create(node_postal_mock);
+
+    var actual = parser.parse('quéry vàlue');
+
+    var expected = {
+      city: 'city value',
+      state: 'state value'
+    };
+
+    t.deepEqual(actual, expected);
```

Comment:
Maybe it doesn't even matter what is returned from the mock parser call. The key check here is line 135, but it's mostly lost due to the huge size of the return value. Could we remove line 15 and then not bother returning anything from node_postal_mock?